toolchain/lexer/tokenized_buffer.cpp

```diff
@@ -540,7 +536,7 @@ class TokenizedBuffer::Lexer {
       // Too short to form one of these tokens.
       return LexResult::NoMatch();
     }
-    if (!('1' <= word[1] && word[1] <= '9')) {
+    if (word[1] < '1' || '9' < word[1]) {
```

Comment:
I find this hard to read, how about:

```suggestion
    if (word[1] < '1' || word[1] > '9') {
```